src/core/Chip/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Icon from '../Icon'
+import s from './styles'
+
+class Chip extends React.PureComponent {
+    onRemove = event => {
+        event.stopPropagation()
+        this.props.onRemove()
+    }
+
+    renderIcon() {
```

Comment:
use a shorter name e.g. `viewIcon`

Reply:
4358c7d, Using functions to show icons if necessary to clean up the render function.

```
{this.showIcon()}
<span className={s('label', { overflow })}>{label}</span>
{this.showRemove()}
```